compiler/rustc_span/src/hygiene.rs

```diff
@@ -645,10 +645,7 @@ pub fn debug_hygiene_data(verbose: bool) -> String {
                 let expn_data = expn_data.as_ref().expect("no expansion data for an expansion ID");
                 debug_expn_data((&id.to_expn_id(), expn_data))
             });
-            // Sort the hash map for more reproducible output.
-            let mut foreign_expn_data: Vec<_> = data.foreign_expn_data.iter().collect();
-            foreign_expn_data.sort_by_key(|(id, _)| (id.krate, id.local_id));
-            foreign_expn_data.into_iter().for_each(debug_expn_data);
+            data.foreign_expn_data.iter().for_each(debug_expn_data);
```

Comment:
If I understand correctly, this is only used for debugging so which order we use shouldn't matter, as long as there is a consistent order, so relying on the hashmaps order should be fine.

Reply:
Yes, any consistent order would work here.